scripts/build-shotcut.sh

```diff
@@ -617,7 +617,7 @@ function set_globals {
     CFLAGS_[0]="${CFLAGS_[0]} -I/opt/local/include"
     LDFLAGS_[0]="${LDFLAGS_[0]} -L/opt/local/lib"
   elif test "$TARGET_OS" = "Linux" ; then
-    CONFIG[0]="${CONFIG[0]} --enable-x11grab --enable-libpulse"
+    CONFIG[0]="${CONFIG[0]} --enable-libxcb --enable-libpulse"
```

Comment:
Apparently this replaces x11grab. But they still use the x11grab device name internally. So it might just work. But since screen grab is currently disabled in Shotcut, maybe it doesn't matter.